source/main.cpp

```diff
@@ -95,6 +96,12 @@ int main(int argc, char *argv[])
 		if(!GameData::BeginLoad(argv))
 			return 0;
 		
+		if(!testToRun.empty() && !GameData::Tests().Has(testToRun))
+		{
+			cout << "Test not found." << endl;
```

Comment:
This should report the test that was asked for, and possibly also print the game's known tests.

Reply:
I expect (or at least hope) the list of known tests to become quite long in the future, so I choose to only print the test-name to ensure that the error-message doesn't get lost by a long scrolling list.